docs/modules/ROOT/pages/reference/billing-naming.adoc

```diff
@@ -0,0 +1,67 @@
+= Naming Scheme for Billing
+
+This page specifies various aspects of the AppCatalog billing.
+
+== Product Naming in the Database
+
+In order to support all future AppCatalog services and its subproducts, we need to define a naming scheme to differentiate between them.
+Each AppCatalog service will have multiple variations, let's take PostgreSQL as an example:
+
+* PostgreSQL standalone without SLA
+* PostgreSQL standalone with SLA
+* PostgreSQL replicated without SLA
+* PostgreSQL replicated with SLA
+* PostgreSQL CloudSQL
+* PostgreSQL RDS
+
+Every permutation will need an identifier, as it will be associated with a product in the https://kb.vshn.ch/appuio-cloud/references/architecture/metering-data-flow.html#_data_model[product dimension] of the billing database.
+Also, depending on the cluster where a given service is running, there could be different prices and discounts.
+
+To accommodate for all these variations, the default identifier of each product in the database should be as follows:
+
+ appcat_$servicename_$architecture_$sla:$clustername
+
+* `servicename`: The name of the AppCatalog service, in this example PostgreSQL
+* `architecture`: The architecture type of the service, for example `standalone` or `rds`.
+* `sla`: The name of the applied SLA.
+* `clustername`: The name of the cluster where this given instance is provisioned.
```

Comment:
"provisioned" or "ordered"? I know we currently don't support service clusters, but maybe it way come later.
```suggestion
* `clustername`: The name of the cluster where this given instance is ordered.
```